mtp_transaction_uploader/upload.py

```diff
@@ -0,0 +1,125 @@
+import os
+import shutil
+import re
+from datetime import datetime
+from collections import namedtuple
+
+from bankline_parser.data_services import parse_file
+from bankline_parser.data_services.enums import TransactionCode
+from pysftp import Connection
+
+from .api_client import get_authenticated_connection
+from . import settings
+
+ref_pattern = re.compile('([A-Za-z][0-9]{4}[A-Za-z]{2}).*?([0-9]{1,2}).*?([0-9]{1,2}).*?([0-9]{2,4})')
+file_pattern = re.compile('YO1A\.REC\.#D\.(%(code)s)\.D([0-9]{6})' % {'code': settings.ACCOUNT_CODE})
```

Comment:
I wonder if it's worth breaking these out using [`re.VERBOSE`](https://docs.python.org/3/library/re.html#re.VERBOSE) and adding some comments to describe the format more clearly?